sources/api/migration/migrations/v1.20.0/static-pods-services-cfg-v0-1-0/src/main.rs

```diff
@@ -0,0 +1,23 @@
+use migration_helpers::common_migrations::{ListReplacement, ReplaceListsMigration};
+use migration_helpers::{migrate, Result};
+use std::process;
+fn run() -> Result<()> {
+    migrate(ReplaceListsMigration(vec![
+        ListReplacement {
+            setting: "services.static-pods.configuration-files",
+            old_vals: &[],
+            new_vals: &[],
```

Comment:
This would be broken on a downgrade -> upgrade cycle.
```suggestion
            old_vals: &[],
            new_vals: &["static-pods-manifest"],
```

Reply:
Flagging this for my own re-review as well.